src/App.scss

```diff
@@ -63,28 +64,29 @@
       padding: 1rem 0.085em;
     }
   }
+
   .App {
     --targetselector-display: unset;
     --sidebar-display: unset;
     --header-display: unset;
     --main-display: unset;
-    grid-template: 'sidebar target-selector main';
```

Comment:
Grid is a nice approach to creating complex layouts with ease. In our situation, I found this approach not quite modular. Every item (like SideBar, AppSelector, Main, etc) should have its sizing properties on its own implementation. This will make them more movable and maintainable

Since the SideBar and AppSelector have static widths, lets handle their styles on their class and just use flex to position them.